specs/corla.pvs

```diff
@@ -217,11 +239,34 @@ BEGIN
 %%% by filling in a single bubble to enumerating a total order on all
 %%% choices in a contest. Across a given ballot, each contest has a
 %%% unique name.
+% @review CDOS will there be one risk limit? If more than one, then
+% system must provide a way to map contests to risk limits.
 
   contest: TYPE
   contest_name?: [contest -> string]
+% @trace RDPR-6-Jul-2017 Section 25.1.2
+% @description A contest selected by the Secretary of State for a
+% risk-limiting audit.
+  audited_contest: TYPE FROM contest
```

Comment:
Getting them to tighten down language, and seeing that reflected in our domain model, is important.  I also noticed that their discussion of hashes and hashing is a mess in the current draft rules, e.g.